### PR TITLE
Create CONTRIBUTING.md

### LINKED ISSUE #383: Add CONTRIBUTING.md
Wishlist 

- [ ] Add variable naming conventions: so far we're tending towards PascalCase for Classes, camelCase for methods and snake_case for other stuff
- [ ] Don't send PRs which reduce the coverage status
- [ ] other stuff (?)

### PR DESCRIPTION
**Pull request by ahmetcetin**
resolve #383

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## Working on GroupButler
+### Prerequisites
+* [Git](https://git-scm.com/)
+
+### Getting GroupButler
+1. Fork a copy of our repo
+2. Open up Git in an environment of your choice
+3. Run the following
+
+```
```
Comment (TiagoDanin) [inline CONTRIBUTING.md]: \`\`\`bash

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+$ git clone https://github.com/YOUR-GITHUB-PROFILE-NAME/GroupButler.git
+$ cd GroupButler
+```
+
+### Please pay attention to
+1. open an issue describing the feature/bug you wish to contribute first to start a discussion, explain why, what and how
+2. follow the eslint rules
```
Comment (TiagoDanin) [inline CONTRIBUTING.md]: eslint ?

**Comment by TiagoDanin** [comment]
- Missing "Don't send PRs which reduce the coverage status"

**Comment by NotAFile** [comment]
>     * Missing "Don't send PRs which reduce the coverage status"

meh. It's often okay. Adding anything is going to reduce the coverage status when coverage is this low, really.

**Comment by yangm97** [comment]
@NotAFile this is the secret. if you add more LoC, you have to add tests to cover them, so coverage does not reduce even further

File: CONTRIBUTING.md (hunk 1 above)
Comment (ahmetcetin) [reply CONTRIBUTING.md]: added

File: CONTRIBUTING.md (hunk 2 above)
Comment (ahmetcetin) [reply CONTRIBUTING.md]: removed

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+### Prerequisites
+* [Git](https://git-scm.com/)
+
+### Getting GroupButler
+1. Fork a copy of our repo
+2. Open up Git in an environment of your choice
+3. Run the following
+
+```bash
+$ git clone https://github.com/YOUR-GITHUB-PROFILE-NAME/GroupButler.git
```
Comment (yangm97) [inline CONTRIBUTING.md]:
```suggestion
$ git clone https://github.com/group-butler/GroupButler.git
```

**Pull request merged**